api/server.spec.js

```diff
@@ -0,0 +1,87 @@
+const server = require('./server');
+const request = require('supertest');
+const db = require('../data/dbConfig');
+
+describe('The server', () => {
+
+  describe('GET /', () => {
+    it('should return status 200', async () => {
+      const res = await request(server).get('/');
+      expect(res.status).toBe(200);
+    })
+
+    it('should return JSON', async () => {
+      const res = await request(server).get('/');
+      expect(res.type).toBe('application/json')
+    })
+
+    it ('should return { api: "Welcome to the Games API!"}', async () => {
+      const res = await request(server).get('/');
+      expect(res.body).toEqual({ api: 'Welcome to the Games API!'})
+    })
+  })
+
+  describe('GET /games', () => {
+    beforeEach(() => {
+      return db('games').truncate();
```

Comment:
You might want to make this an `async...await` function just in case. It seems that currently it isn't causing problems, but, the more data you start working with/the larger tests you have, the more likely you're to run into trouble with this asynchronous code. To ensure that this finishes before each test:
```javascript
beforeEach(async () => {
  await db('games').truncate();
})
```